lib/puma/metrics/parser.rb

```diff
@@ -46,6 +46,10 @@ def register_default_metrics # rubocop:disable Metrics/MethodLength
                        docstring: 'Maximum number of worker threads',
                        labels: [:index],
                        preset_labels: { index: 0 })
+        registry.gauge(:puma_requests_count,
```

Comment:
Should this not be a counter instead? 

https://prometheus.io/docs/concepts/metric_types/#counter